{{cookiecutter.project_name}}/.env.docker

```diff
@@ -0,0 +1,2 @@
+UID="$(id -u)"
```

Comment:
Can you please explain why do we need this file?

Reply:
We need to uid and gid from host. docker-compose does not suppor shell command. You can see issue about it: [https://github.com/docker/compose/issues/4081](https://github.com/docker/compose/issues/4081)

We have some options to do this.
1.
`bash
export UID=$(id -u)
export GID=$(id -u)
docker-compose -f docker-compose.yml -f docker-compose.override.yml up --build
`

2.
`bash
UID=$(id -u) GID=$(id -g) docker-compose -f docker-compose.yml -f docker-compose.override.yml up --build
`
3
`
docker-compose -f docker-compose.yml -f docker-compose.override.yml --env-file ./.env.docker up --build
`

We can choose one of them.

Reply:
Can we use the existing `.env` file? It's inside the `config/` dir

Reply:
Yes actually we can. But docker looks .env file in root directory and read automatically. But I think we can not run bash command in env file. So ı deleted it.